### PR TITLE
Update naming of CE

### PR DESCRIPTION
**Pull request by rod-glover**
Following Francis' requested changes:

About   CE

About   Credits

### CODE REVIEW

**Comment by rod-glover** [comment]
@corviday , would you mind verifying these changes against these requested changes?

[PCEX_About_FZ_edits.pdf](https://github.com/pacificclimate/climate-explorer-frontend/files/2958930/PCEX_About_FZ_edits.pdf)

File: src/components/guidance-content/about/ClimateExplorer.js [hunk 1]
```diff
@@ -17,11 +17,11 @@ export default function ClimateExplorer() {
...
               <p>
                 A tool for visualizing and downloading
-                climate model data and data derived from climate outputs
-                for the B.C. - Yukon region.
+                climate model outputs and data derived from those outputs.
               </p>
               <p>
-                Climate Explorer is also fondly known as 'the Marmot,'
+                The PCIC Climate Explorer (PCEX) is also fondly known as
+                the 'marmot,'
```
Comment (corviday) [inline src/components/guidance-content/about/ClimateExplorer.js]: An unrequested (though trivial) change you've made:

`'the marmot'` 
to 
`the 'marmot'`

Comment (rod-glover) [reply src/components/guidance-content/about/ClimateExplorer.js]: It was intentional; it's one of those trad vs. modern things. Trad says punctuation goes inside quotes. I'm kind of a trad guy, but I really don't care very much.

Comment (corviday) [reply src/components/guidance-content/about/ClimateExplorer.js]: I  certainly agree with you on that! The difference I was commenting on is that "the" was previously inside the quotes and is now outside them.

**Pull request merged**